src/datasource.js

```diff
@@ -156,25 +82,79 @@ export class GenericDatasource {
     });
   }
 
-  selectQueryFunc() {
+  suggestTags(type, key) {
+    if (!key) {
+      // Need at least some characters typed in order to suggest something
+      return this.q.when([]);
+    }
     return this.backendSrv.datasourceRequest({
-      url: this.url + '/status',
+      url: this.url + '/' + type + 's/tags/' + key + ':*',
```

Comment:
This will only return tags which keys fully match the key param, correct? Meaning, if you type "memo" in the tag field, you won't get "memory:xxx" tag suggestions?